events-manager/src/main/java/metcamp/backend/eventsmanager/service/ValidationService.java

```diff
@@ -0,0 +1,135 @@
+package metcamp.backend.eventsmanager.service;
+
+
+import metcamp.backend.eventsmanager.entities.model.*;
+import metcamp.backend.eventsmanager.exceptions.ValidationException;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class ValidationService {
+
+    public void validateCreateEvent(Event event){
+        validateId(event.getId());
+        validateName(event.getName());
+        validateDates(event.getStartDateTime(), event.getEndDateTime());
+        validateAttendeess(event.getAttendees());
+        validateOrganizer(event.getOrganizer());
+        validateEventType(String.valueOf(event.getEventType()));
+        validatePrices(event.getPrices());
+    }
+
+
+    public void validateName (String name) {
+        //Validar si el nombre no esta vacio y que tenga al menos 5 caracteres
+        if (name.isEmpty() || name.isBlank()){
+            throw new ValidationException("name is required");
+        }
+        if (name.length() < 5){
+            throw new ValidationException("name is too short");
+        }
+    }
+
+    public void validateId(int id){
+        if (id==0){
+            throw new ValidationException("id must not be zero");
+        } else if (id < 0){
+            throw new ValidationException("id must be positive");
+        }
+    }
+
+    public void validateDates(LocalDateTime startDate, LocalDateTime endDate){
+        //Verificar que start sea antes que end
+        if (startDate.isAfter(endDate)){
+            throw new ValidationException("startDate must be before endDate");
+        }
+    }
+
+    public void validateAttendeess(int cantidad){
+        if (cantidad==0){
+            throw new ValidationException("attendees must not be zero");
+        } else if (cantidad < 0){
+            throw new ValidationException("attendees must be positive");
+        }
+    }
+
+    public void validateOrganizer (String org) {
+        if (org.isEmpty() || org.isBlank()){
```

Comment:
Si miramos la doc:
- isEmpty(): Returns true if, and only if, length() is 0. --> es decir da true si la cadena está vacía
- isBlank(): Returns true if the string is empty or contains only white spaces --> es decir da true si la cadena está vacía o tiene sólo espacios en blanco

esto implica que usar ambas validaciones es redundante porque isBlank() ya incluye la validación de isEmpty()
 
<img width="379" alt="image" src="https://github.com/napoli-lucia/Proyecto-Backend-Java/assets/108366965/bd7d5d58-06dc-4cd9-a496-e6df016aacd2">